### PR TITLE
add RU

### PR DESCRIPTION
**Pull request by danchousz**
ENS: danch.eth
Mail: d.potyomka@gmail.com

### CODE REVIEW

File: public/locales/ru/translations.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
+{
+  "language name": "Русский",
+
+  "home": "Главная",
+  "recent": "Недавняя Активность",
+  "no followers": "Ни одного подписчика",
+  "latest followers": "Последние Подписавшиеся",
+  "connect wallet to view": "Подключите кошелек, чтобы увидеть",
+  "phrase": "Все о тех, кого вы знаете. 😜",
```
Comment (danchousz) [inline public/locales/ru/translations.json:9]: The phrase "It's about who you know." in Russian sounds silly. Made some minor changes, now it sounds like "All About People You Know".

File: public/locales/ru/translations.json [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  "social graph first": "Социальная сеть",
```
Comment (danchousz) [inline public/locales/ru/translations.json:10]: Social graphs is a phrase that is not used in everyday life in Russian. Instead, "social network" is used. The concept is broader, but very common.

File: public/locales/ru/translations.json [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+  "follower list second": "которым Вы владеете.",
+  "integrations": "Интеграции",
+  "partners": "Нас поддержали",
+  "sponsors": "Также с помощью грантов:",
+  "support": "Служба поддержки",
+
+  "profile": "Мой Профиль",
+  "profile error": "Такого пользователя не существует.",
+  "settings": "Настроить список",
+  "block-mute": "Заблокировать/Замьютить список",
```
Comment (danchousz) [inline public/locales/ru/translations.json:24]: The word "Mute" has a literal translation in Russian, and it sounds like Заглушить (Zaglushit'). The word is obsolete and is not used; instead, the transliteration-Anglicism Замьютить (Zamutit') is used, which is a direct tracing of the word "Mute".

**Comment by brantlymillegan** [comment]
Thanks so much, but we got a good Russian translation earlier today already. Feel free to check it out once we deploy it and submit corrections and improvements

File: public/locales/ru/translations.json [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+  "edit settings": "Править Настройки",
+  "edit profile": "Редактировать",
```
Comment (danchousz) [inline public/locales/ru/translations.json:26]: The same word edit is translated in one case as Править (Pravit'), which refers more to settings, and in another case as Редактировать (Redaktirovat'), which refers to personal information.

File: public/locales/ru/translations.json [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+  "top eight title": "🔥 Мой Топ-8 🔥",
```
Comment (danchousz) [inline public/locales/ru/translations.json:27]: Top X Usually written with a dash in Russian interfaces.

**Comment by danchousz** [comment]
Looked at this guy's translation. It looks like a slightly edited machine translation. I can give a review of his translation if needed

**Comment by brantlymillegan** [comment]
> Looked at this guy's translation. It looks like a slightly edited machine translation. I can give a review of his translation if needed

hey I DMed you on twitter/x

File: public/locales/ru/translations.json (hunk 2 above)
Comment (danchousz) [reply public/locales/ru/translations.json:10]: Got it. In the Russian marketing school, there is a notion that with Russians it is always worth using the most general concepts. However, if you also want to educate people, the term social graph will do just fine.

**Pull request closed without merging**